compiler/rustc_codegen_llvm/src/allocator.rs

```diff
@@ -132,9 +140,8 @@ fn create_wrapper_function(
             None
         };
 
-        if tcx.sess.default_hidden_visibility() {
-            llvm::LLVMRustSetVisibility(llfn, llvm::Visibility::Hidden);
-        }
+        apply_visibility(tcx.sess.default_visibility(), llfn);
```

Comment:
What's the reason for adding and using `apply_visibility` instead of just doing `llvm::LLVMRustSetVisibility(llfn, Visibility::from_generic(self.tcx.sess.default_visibility()));` ?

Reply:
I suspect I did that because I wanted to preserve not calling `llvm::LLVMRustSetVisibility` when the visibility was `default`, but that's probably not a very good reason, so I've changed it as you suggested.